source/bayes_simulation.Rmd

```diff
@@ -249,102 +507,141 @@ One may obtain an answer as follows:
-    data) seven black offspring. One can then calculate, as Fisher
-    (1959, p.17) did, the probabilities, posterior to the data, of the
-    test mouse being homozygous ( *BB* ) and heterozygous ( *Bb* ) using
-    Bayes' theorem. . .
-
-    We see that, given the genetic characteristics of the offspring, the
-    mating results of 7 black offspring changes our knowledge
-    considerably about the test mouse being ( *BB* ) or ( *Bb* ), from a
-    prior probability ratio of 2:1 in favor of ( *Bb* ) to a posterior
-    ratio of 64:1 against it (1973, pp. 12-14).
-
-    1.  Let us begin, as do Box and Tiao, by restricting our attention
-        to the third line in Table 25-1, and let us represent those
-        results with 4 balls---1 black with "BB" painted on it, 2 black
-        with "Bb" painted on them, and 1 brown which we immediately
-        throw away because we are told that the "test mouse" is black.
-        The remaining 3 (black) balls are put into a bucket labeled
-        "test."
-
-        Table 25-1
-
-        **Probabilities for Genetic Character of Mice Offspring**
-
-        **Mice BB (black) Bb (black) bb (brown)**
-
-          ------------------ --- --- ---
-          BB mated with bb   0   1   0
-          Bb mated with bb   0   ½   ½
-          Bb mated with Bb   ¼   ½   ¼
-          ------------------ --- --- ---
-
-         Source: Box and Tiao, 1973, pp. 12-14
-
-    2.  From prior knowledge we know that a BB black mouse mated with a
-        bb brown mouse will produce all black mice (line 1 in the
-        table), and a Bb black mouse mated with a bb brown mouse will
-        produce 50 percent black mice and 50 percent brown mice. We
-        therefore construct two more buckets, one with a single black ball
-        (the bucket labeled "BB") and the other with one black ball and one
-        brown ball (the bucket labeled "Bb"). We now have three buckets.
-
-    3.  Take a ball from bucket "test." If its label is "BB," record that
-        fact, take a ball (the only ball, which is black) from the BB
-        bucket, record its color (we knew this already), and replace the
-        ball into the BB bucket; the overall record of this trial is
-        "BB-black." If the ball drawn from bucket "test" says "Bb," draw a
-        ball from the Bb bucket, record, and replace; the record will
-        either be "Bb-black" or "Bb-brown."
-
-    4.  Repeat step 3 seven times.
-
-    5.  Examine whether the record of the seven balls drawn from the BB
-        and Bb buckets are all black; if so, record "Y," otherwise "N."
-
-    6.  Repeat steps 3-5 perhaps 1000 times.
-
-    7.  Ignore all "N" records. Proceeding now if the result of step 5
-        is "Y": Count the number of cases which are BB and the number
-        which are Bb. The proportions of BB/"Y" and Bb/"Y" trials are
-        the probabilities that the test mouse is BB and Bb respectively.
+## Fundamental problems in statistical practice
```

Comment:
Yeah - I don't know about this one.  It seems to me that Simon is just trying to show that a well-know textbook Bayes problem fits with the resampling idea - that his audience is readers of various textbooks rather than students!